companion/delphilint-vscode/src/extension.ts

```diff
@@ -1,26 +1,27 @@
-/*
- * DelphiLint VSCode
- * Copyright (C) 2024 Integrated Application Development
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program. If not, see <https://www.gnu.org/licenses/>.
- */
+/*
```

Comment:
What happened here, did something flip the line endings?

Reply:
Huh, I'm really not sure. Maybe something to do with the license header script?

Reply:
Looks like `extension.ts` was incorrectly LF instead of CRLF.  Not sure why...

Reply:
I didn't add one at the start and I haven't got around to doing it here (bad, I know). The line endings are generally consistently CRLF.